lib/inputs/pixel-trackers.js

```diff
@@ -0,0 +1,103 @@
+const crypto = require('crypto')
+const logger = require('../logger')
+const timestamp = require('../timestamp')
+const lookip = require('../lookip')
+const bigquery = require('../bigquery')
+
+/**
+ * Pixel.prx.org trackers
+ */
+module.exports = class PixelTrackers {
+
+  constructor(records) {
+    this._records = (records || []).filter(r => this.check(r))
+  }
+
+  check(record) {
+    return record.type === 'pixel'
+  }
+
+  async insert() {
+    if (this._records.length === 0) {
+      return []
+    }
+
+    // format records and organize by dataset + table
+    const formatted = await Promise.all(this._records.map(r => this.format(r)))
+    const grouped = this._records.reduce((acc, rec, idx) => {
+      const dest = this.destination(rec)
+      if (dest) {
+        (acc[dest] = acc[dest] || []).push(formatted[idx])
+      }
+      return acc
+    }, {})
+
+    // insert in parallel
+    return await Promise.all(Object.keys(grouped).map(async (datasetTable) => {
+      try {
+        const [ds, tbl] = datasetTable.split('.')
+        const num = await bigquery.insert(ds, tbl, grouped[datasetTable])
+        return {count: num, dest: datasetTable}
+      } catch (err) {
+        if (err.code === 404) {
+          logger.error(`Table not found: ${datasetTable}`)
+          return {count: 0, dest: datasetTable}
+        } else {
+          const insertErrors = logger.combineErrors(err)
+          if (insertErrors) {
```

Comment:
This effectively ignores (and logs) insert errors.  Which are normally schema issues, or a missing required field (generally we only require `timestamp`).